cdc/sinkv2/tablesink/state/state.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package state
+
+import "sync/atomic"
+
+// TableSinkState is state of the table sink.
+type TableSinkState int32
+
+// State for table sink
+const (
+	TableSinkStateUnknown TableSinkState = iota
```

Comment:
```suggestion
	TableSinkStateUnknown TableSinkState = iota+1
```
Make zero values panic.

Reply:
Is this a best practice? Why do we need this? Could you please explain a little bit?